test/e2e/steps/see_dialog_position.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright © 2022-2025 IT ENGINEERING MANAGEMENT INC,
+//                       <https://github.com/team113>
+//
+// This program is free software: you can redistribute it and/or modify it under
+// the terms of the GNU Affero General Public License v3.0 as published by the
+// Free Software Foundation, either version 3 of the License, or (at your
+// option) any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License v3.0 for
+// more details.
+//
+// You should have received a copy of the GNU Affero General Public License v3.0
+// along with this program. If not, see
+// <https://www.gnu.org/licenses/agpl-3.0.html>.
+
+import 'package:get/get.dart';
+import 'package:gherkin/gherkin.dart';
+import 'package:messenger/domain/model/chat.dart';
+import 'package:messenger/ui/page/home/tab/chats/controller.dart';
+
+import '../parameters/position_status.dart';
+import '../world/custom_world.dart';
+
+/// Indicates whether a dialog [Chat] with the provided name is displayed at the
+/// specified [PositionStatus].
+///
+/// The [Chat] object represents a dialog between users.
+///
+/// Examples:
+/// - Then I see "Bob" dialog first in favorites list
+/// - Then I see "Bob" dialog last in favorites list
+final StepDefinitionGeneric seeFavoriteDialogPosition =
+    then2<String, PositionStatus, CustomWorld>(
+      'I see {string} dialog {position} in favorites list',
```

Comment:
Since `Bob` is a name, shouldn't `TestUser` variable be used there instead of `String`? This way the line may become something like that:

```
Then I see dialog with Bob first in favorites list
```

```suggestion
/// - Then I see "Bob" dialog first in favorites list
/// - Then I see "Bob" dialog last in favorites list
final StepDefinitionGeneric seeFavoriteDialogPosition =
    then2<TestUser, PositionStatus, CustomWorld>(
      'I see dialog with {user} {position} in favorites list',
```